src/autocomplete/databases/clickhouse/grammar/ClickHouseParser.g4

```diff
@@ -998,9 +997,13 @@ insertFormatType
     ;
 
 dataClause
-    : FORMAT insertFormatType identifier+ # DataClauseFormat
-    | FORMAT? valuesStatement             # DataClauseValues
-    | selectUnionStatement SEMICOLON? EOF # DataClauseSelect
+    : FORMAT insertFormatTypeExceptValues any
+    | FORMAT? valuesStatement
+    | selectUnionStatement SEMICOLON? EOF
+    ;
+
+any
```

Comment:
nit: I kind of don't like this name
maybe better to call it `anyValue` or `anyToken`?